CMakeLists.txt

```diff
@@ -1,11 +1,36 @@
 # CMake4GDAL project is distributed under MIT license. See accompanying file LICENSE.txt.
+# Modified by Mi Zhang, LuoJiaNET Development and Research Group, Wuhan University, 2022
+
 cmake_minimum_required(VERSION 3.9...3.23)
 
 project(gdal LANGUAGES C CXX)
 include(CTest)
 
 set(GDAL_LIB_TARGET_NAME GDAL)
 
+###############################################################################
```

Comment:
that logic should be moved to swig/python/CMakeLists.txt

Reply:
Yes, it is more appropriate to keep it in  swig/python folder.